www/docs/server/fastify.md

```diff
@@ -157,18 +157,18 @@ Your endpoints are now available via HTTP!
 
 ## How to enable subscriptions (WebSocket)
 
-The Fastify adapter supports [subscriptions](subscriptions) via the [fastify-websocket](https://www.npmjs.com/package/fastify-websocket) plugin. All you have to do in addition to the above steps is install the dependency, add some subscriptions to your router and activate the `useWSS` [option](#fastify-plugin-options) in the plugin.
+The Fastify adapter supports [subscriptions](subscriptions) via the [@fastify/websocket](https://www.npmjs.com/package/@fastify/websocket) plugin. All you have to do in addition to the above steps is install the dependency, add some subscriptions to your router and activate the `useWSS` [option](#fastify-plugin-options) in the plugin.
```

Comment:
@lukahartwig could you add a note here about minimum fastify version?

Reply:
I add a sentence about the minimum fastify version that's required for `@fastify/websocket` as well as the last version of `fastify-websocket`.